### ISSUE TITLE
External resquests on Tutorial JavaScript REST API result in 400 Error: Object reference not set to an instance of an object.

### ISSUE THREAD

**Issue by 10PoundGorilla-ChrisBehling** [2sic/2sxc#2159]
**I'm submitting a ...** 
[x] bug report

**...about**
[x] APIs like REST

**Current behavior**
Using the template example number 11 (_11 List, create and delete using REST.cshtml) to use an external POST to send data to the 2sxc components API results in a **400 error message: Object reference not set to an instance of an object**. 

**Expected behavior**
Posting data to API creates new element.

**Minimal reproduction of the problem with instructions**
- Install 2sxc
- Install 2sxc app tutorial (https://2sxc.org/en/apps/app/tutorial-javascript-rest-api-using-jquery-and-angularjs)
- Add app to page
- Select template 11
- Click to create external POST request 

**Please tell us about your environment:**
- Tested on DNN 9.6.0 (2sxc version 11.2.0) and DNN 9.1.0 (2sxc version 9.32.1)
- Tested with Host user, Administrator, and unauthenticated (should fail for unauthenticated by default though)
- Tested on Firefox and Chrome

**Comment by iJungleboy** [comment]
thx, we'll check into this. which 2sxc? you didn't mention...

**Comment by 10PoundGorilla-ChrisBehling** [comment]
@iJungleboy I updated my original details: DNN 9.6.0 (2sxc version 11.2.0) and DNN 9.1.0 (2sxc version 9.32.1)

**Comment by iJungleboy** [comment]
I just ran some tests and all #11 worked. 

The only thing that failed was #23 angular-query (it has the wrong query name), but 11 worked.

Tested on both DNN 7.4.2 and 9.x with 2sxc 11.05. 

Anything to go by why your example failed? could you write what urls it was using?

I'll just republish the app now - but just fixing the old angular example

**Comment by 10PoundGorilla-ChrisBehling** [comment]
@iJungleboy, I tried recording a video of the issue we were running into and I wanted to go as far as completely uninstall 2sxc. But after a fresh install with both the module and demo 2sxc app; the code is working as expected. I did end up installing using 2sxc 11.05.00 and the App_Tutorial-Java... version 8.11.0. 

I tested deleting the demo 2sxc app and reinstalling on our other development environment and after reinstalling the demo app we are still experiencing the same issue with external requests. 

I then installed the latest release of 2sxc (11.05.00) and external api posts worked in this environment. There must have been some conflict with the versions of 2sxc that we had on these environments, or maybe it was resolved in more recent updates.

As noted we were originally testing on 2sxc versions: 9.32.1 and 11.2.0.

I think the most recent release of 2sxc we have on a production or near-production site is 11.2.0, but most are in late 9's or 10's.
